README.md

```diff
@@ -0,0 +1,35 @@
+First off, we are using Linux because our software only works on Linux right now (If our software supports other operating systems, eg. Windows and MacOS, please ignore this readme). If you already have Linux install, plesase disregard this readme. We recommend two options if you do not have Linux already.
+
+# First option is dual booting [Highly Recommended]
+Background Knowledge: Dual booting will keep 2 operating systems on your machine (only one operating system will be running on your machine at a time), so when you first start your computer, you can choose which operating system you want to run on.
+We highly recommend dual booting since you can utilize 100% of your computer's hardware for only one operating system(the second option, using a Virtual Machine, shares the hardware of PC between the two operating systems; look at VM section for more information). The pro about dual booting is that the computer's performance is maximized since it only needs to focus on the one operating system, but the con about dual booting is that if you want to use your other operating system, you have to restart your computer. 
+
+For both dual booting and installing Linux on a VM, get a Debian distro (Linux) of your choice.
+   * Recommended distros are:
+   		Linux Mint (Latest Version)
+   		Ubuntu (Latest Version)
+
+How to Dual Boot:
+Follow the instructions and/or watch the videos below.
+For Windows
+Written Instructions: https://itsfoss.com/install-ubuntu-1404-dual-boot-mode-windows-8-81-uefi/ 
+Video: https://www.youtube.com/watch?v=MSVV_EoApdo
+
+For Macs:
+Written Instructions: https://www.howtogeek.com/187410/how-to-install-and-dual-boot-linux-on-a-mac/
+Video: https://www.youtube.com/watch?v=kRgKlcm1XPI
+
+
+# Second option is Running Linux in a Virtual Machine
+Background Knowledge: A virtual machine allows two operating systems to run simulanteously (Sharing the computer's hardware). Using a virutal machine separates the host machine and the simulated machine, so if you are running linux in the virtual machine and windows as your main OS, they will have no idea that each other exist. The pro about using a virtual machine is that you can switch between the operating systems by simply minimizing/maximizing the virtual machine, but the con about using a virtual machine is that the performance of your machine will be reduced (Makes working on projects difficult). If you have a powerful PC and want to take the risk of the performance reduction, feel free to use this option.
```

Comment:
Misspelled virtual as "virutal"

Reply:
Simultaneously*

Reply:
“They will have no idea that the other exists” makes more sense I think